config/routes.rb

```diff
@@ -15,7 +15,8 @@
       resources :orders
     end
     resources :users, only: %i(new show create) do
-      resources :orders, only: %i(new index create)
+      get :orders, to: "orders#user_index"
+      resources :orders, only: %i(new create)
```

Comment:
Sao cần phải có line này nhỉ, dùng ở line 18 nhỉ, với lại sao phải tạo ra cái tên `user_index` vậy?
chỉ cần 19:` resources :orders, only: %i(new create index)`
```suggestion
       resources :orders, only: %i(new create index)
```

Reply:
ah chú tạo user_index để khác với index (shop) chứ j

cái này thi em tách ra nhé

refactor lại:

order của shop thì nằm ở namspace shop